src/identity/auth.ts

```diff
@@ -51,11 +52,11 @@ export class Auth {
 		let strategy: oauth2.Strategy
 		strategy = new oauth2.Strategy(
 			{
-				authorizationURL: `${this.config.authenticationServiceUrl}/oauth/authorize`,
+				authorizationURL: `${this.config.authenticationServiceUrl}` + config.AUTHENTICATION.oauthAuthorizeEndpoint,
```

Comment:
use either + or templates (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here, not both